<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGUIClass</class>
 <widget class="QMainWindow" name="QtGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="weatherData"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="precipitationEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="showAllButton">
          <property name="text">
           <string>Show All</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="precipitationButton">
          <property name="text">
           <string>Filter by Precipitation</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="descriptionEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="descriptionButton">
          <property name="text">
           <string>Get Total Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timeData">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGUI.qrc"/>
 </resources>
 <connections/>
</ui>
